subprojects/ide-native/src/integTest/groovy/org/gradle/ide/xcode/fixtures/AbstractXcodeIntegrationSpec.groovy

```diff
@@ -191,4 +191,12 @@ Actual: ${actual[key]}
         assert target.buildConfigurationList.buildConfigurations[0].buildSettings.PRODUCT_NAME == expectedProductName
         assert target.buildConfigurationList.buildConfigurations[0].buildSettings.SWIFT_INCLUDE_PATHS == null
     }
+
+    void assertTargetIsIndexerWithTestComponent(ProjectFile.PBXTarget target, String expectedProductName) {
+        assert target.productName == expectedProductName
+        assert target.name.startsWith("[INDEXING ONLY] $expectedProductName")
+        assert target.buildConfigurationList.buildConfigurations.name == [DefaultXcodeProject.BUILD_DEBUG]
+        assert target.buildConfigurationList.buildConfigurations[0].buildSettings.PRODUCT_NAME == expectedProductName
+        assert target.buildConfigurationList.buildConfigurations[0].buildSettings.SWIFT_INCLUDE_PATHS == '"' + file('build/modules/main').absolutePath + '"'
```

Comment:
We want to assert `SWIFT_INCLUDE_PATHS` in `assertTargetIsUnitTest` for all configuration. The indexer for XCTest in Xcode doesn't do anything. It's all thanks to setting `SWIFT_INCLUDE_PATHS` for all three build configuration that the indexing work properly. Something like the following in `assertTargetIsUnitTest` method should be good enough:

```
assert target.buildConfigurationList.buildConfigurations.every {
    it.buildSettings.SWIFT_INCLUDE_PATHS == '"' + file('build/modules/main').absolutePath + '"'
}
```

Reply:
Is it wrong for the "Indexer" targets to have anything in their `SWIFT_INCLUDE_PATHS`? This test was previously failing because they were expected to be null, but actually had values.

Reply:
It's not wrong to have something in `SWIFT_INCLUDE_PATHS`. It's probably some leftovers from when I extracted those assertion methods. You can probably modify it to pass a list of what is expected or something for `SWIFT_INCLUDE_PATHS`.

Reply:
Done.